src/testing/GlobalFixtures.cpp

```diff
@@ -47,7 +47,7 @@ std::string filterFromLogLevel(boost::unit_test::log_level logLevel)
   namespace bu = boost::unit_test;
 
   if (logLevel == bu::log_successful_tests || logLevel == bu::log_test_units) {
-    return "%Severity% >= debug";
+    return "";
   }
```

Comment:
Why this change?

Reply:
I was facing a (quite intricate) bug and our previous configuration did not provide any option to inspect trace logging from out tests. We should either add an option to get additional trace logging from our test suite or (what this does) add the trace logging by default to the most verbose DEBUG log stream boost test provides.

Reply:
Why couldn't you get trace output from our tests? For me, this always worked in the past via `log.conf`.

Reply:
Using a CLI (from `ctest`) still sounds more easy to me.